examples/c/ex_all.c

```diff
@@ -765,20 +812,40 @@ session_ops(WT_SESSION *session)
 	error_check(session->verify(session, "table:mytable", NULL));
 	/*! [Verify a table] */
 
-	/*! [Drop a table] */
-	error_check(session->drop(session, "table:mytable", NULL));
-	/*! [Drop a table] */
-	}
+	/* Call other functions, where possible. */
+	(void)backup;
```

Comment:
I'll add a better comment: we can't call `backup` because it has absolute paths in the calls, so won't work unless certain directories exist.  We need to do something with the function now it is `static` or the build will fail, so that's what is happening here.